lua/metrics_helper.lua

```diff
@@ -11,7 +11,7 @@ local function get_system_dimension(name)
 end
 
 local default_dimensions = {
-    {"habitat", get_system_dimension('habitat')},
+    {"habitat", tostring(get_system_dimension('habitat'))},
```

Comment:
Because in the tests this returns `nil` since I'm no longer overriding `io.open`.  I'll get rid of this in the future when I move also this file to use configs rather than env variables